### PR TITLE
Monitoring threshold adjustment

### PR DESCRIPTION
**Pull request by yrong**
Resolves: https://linear.app/snowfork/issue/SNO-1067

### CODE REVIEW

**Review by claravanstaden — APPROVED** [review]

File: web/packages/api/src/status.ts [hunk 1]
```diff
@@ -72,8 +72,8 @@ export const BlockLatencyThreshold = {
 }
 
 export const InsufficientBalanceThreshold = {
-    // Minimum as 10 DOT
-    Substrate: 100_000_000_000,
+    // Minimum as 100 DOT
+    Substrate: 1_000_000_000_000,
```
Comment (alistair-singh) [inline web/packages/api/src/status.ts:76]: Let's make it 300 DOT.

**Review by alistair-singh — CHANGES_REQUESTED** [review]

File: web/packages/api/src/status.ts [hunk 2]
```diff
@@ -72,8 +72,8 @@ export const BlockLatencyThreshold = {
...
     // Minimum as 0.1 Ether
     Ethereum: 100_000_000_000_000_000,
```
Comment (alistair-singh) [inline web/packages/api/src/status.ts:78]: I changed all the tenderly alerts to alert if lower than 0.3 ETH. https://linear.app/snowfork/issue/SNO-1067#comment-d4b0c3c5

Maybe we want to do the same here.

File: web/packages/api/src/status.ts (hunk 1 above)
Comment (yrong) [reply web/packages/api/src/status.ts]: https://github.com/Snowfork/snowbridge/pull/1249/commits/3c60b685a6024edb5e377a51bdcaf87fb0118d18

File: web/packages/api/src/status.ts (hunk 2 above)
Comment (yrong) [reply web/packages/api/src/status.ts]: https://github.com/Snowfork/snowbridge/pull/1249/commits/5c0137ed53f9ccc67d73730c0de626a1d3f5d6d2

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**